src/index.js

```diff
@@ -54,7 +54,15 @@ commander
 		"Update lightning branch and create a PR to develop."
 	);
 
-commander.on("--help", () => help());
+commander.on("--help", () => {
+	help();
+
+	if (commander.verbose) {
+		const path = require("path").resolve(__dirname, "workflow.txt"); // eslint-disable-line
```

Comment:
what is this eslint-disable-line for?

also, i know you grabbed this line elsewhere, but it prob would be better to import the path at the top of the file